src/main/java/no/nav/tag/finnkandidatapi/kandidat/KandidatController.java

```diff
@@ -0,0 +1,45 @@
+package no.nav.tag.finnkandidatapi.kandidat;
+
+
+import no.nav.security.oidc.api.Protected;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Protected
+@RestController("/kandidater")
+public class KandidatController {
+
+    private final KandidatRepository kandidatRepository;
```

Comment:
Jeg synes vi burde dele det inn slik: Controller -> Service -> Repository. 
Vi kan lage en metode i KandidatService som heter hentKandidat. Da kan vi fjerne kandidatRepository som avhengighet herfra. Da blir det mindre mocking i testene + at vi hindrer spaghettikode med avhengigheter på kryss og tvers.

Reply:
Riktig. Controlleren burde være så "enkel som mulig" og bare ha ansvar for å ta i mot request og returnere responsen. Jeg fikser.